packages/dx-react-grid/docs/reference/table-header-cell.md

```diff
@@ -0,0 +1,11 @@
+# TableHeaderCell Component Reference
+
+A component that represents the table header cell and is designed to be used as the default `headerCellTemplate` of the [TableHeaderRow](../reference/table-header-row.md) plugin.
+
+## User Reference
+
+### Properties
+
+The default component properties conform the [HeaderCellArgs](../reference/table-header-row.md#header-cell-args).
+
+All the properties specified will be applied to the root header cell.
```

Comment:
May be write "all the rest properties will be spread to the root element"?

Reply:
I think so.